modules/ROOT/pages/install-self-managed-network-configuration.adoc

```diff
@@ -9,7 +9,9 @@ Before installing or using Runtime Fabric on Self-Managed Kubernetes, ensure tha
 
 == Port Configuration
 
-To install or run Runtime Fabric, ensure that you have configured the following ports on your Kubernetes installation:
+To install or run Runtime Fabric, ensure that you have configured the following ports on your Kubernetes installation.
+
+If your firewall permits it, consider allowing the following port traffic to use AMQP (Advanced Message Queuing Protocol) over Websockets.
```

Comment:
can we use "for protocols aware firewalls" instead of "If your firewall permits it". rest looks good to me. Ty